docs/rules/no-todo.md

```diff
@@ -0,0 +1,41 @@
+# Disallow QUnit.todo (no-todo)
+
+`QUnit.todo` is useful for testing code that is under development. However, leaving tests as todo in perpetuity is a bad practice, as the test ceases to provide any use in ensuring correctness of your code. Marking tests as todo should be done sparingly.
+
+## Rule Details
+
+The following patterns are considered warnings:
```

Comment:
It looks like `QUnit.test.todo` / `test.todo` are not covered/mentioned/tested by this rule yet.

Reply:
Hey bmish, I added a test case for [Qunit.test.todo](https://github.com/platinumazure/eslint-plugin-qunit/pull/218/files#diff-249b82d082f5a5b5bcc4d80355c9601df4aa5eb5d98868d386c1b5a5c8baddceR44) and for [test.todo](https://github.com/platinumazure/eslint-plugin-qunit/pull/218/files#diff-249b82d082f5a5b5bcc4d80355c9601df4aa5eb5d98868d386c1b5a5c8baddceR65) 

I added their [documentation](https://github.com/platinumazure/eslint-plugin-qunit/pull/218/files#diff-f40f91961040e27e4d35fa04263030c652b500ed6d2815c41cb7b78c38ce28a5R13) as well

Reply:
Note: I didn't managed to make the (test.todo) work. Should be ok once the #221 will be merged